### PR TITLE
Reorganization of examples 1-3 with minor comment and code edits

### PR DESCRIPTION
**Pull request by niall-turbitt**
- Reorganized examples into a folder structure
- Comments added throughout examples 1-3
- Minor code edits made throughout

### CODE REVIEW

**Comment by niall-turbitt** [comment]
added driver notebook for example 4

File: M1_Sample_Code/1_hello_world/1_hello_world_chain.py [hunk 1]
```diff
@@ -1,19 +1,21 @@
 # Databricks notebook source
 # DBTITLE 1,Install RAG Studio packages
-# MAGIC %run ./wheel_installer
+# MAGIC %run ../wheel_installer
 
 # COMMAND ----------
 
 # Before logging this chain using the driver notebook, you need to comment out this line.
-dbutils.library.restartPython() 
+# dbutils.library.restartPython()
```
Comment (epec254) [inline M1_Sample_Code/1_hello_world/1_hello_world_chain.py]: should we leave this uncommented by default so the notebook runs?

File: M1_Sample_Code/2_hello_world_parameterized/2_hello_world_parameterized_chain.py [hunk 2]
```diff
@@ -1,18 +1,20 @@
 # Databricks notebook source
 # DBTITLE 1,Install RAG Studio packages
-# MAGIC %run ./wheel_installer
+# MAGIC %run ../wheel_installer
 
 # COMMAND ----------
 
 # Before logging this chain using the driver notebook, you need to comment out this line.
-dbutils.library.restartPython() 
+# dbutils.library.restartPython()
```
Comment (epec254) [inline M1_Sample_Code/2_hello_world_parameterized/2_hello_world_parameterized_chain.py]: Same comment - should we leave uncommented?

File: M1_Sample_Code/2_hello_world_parameterized/configs/2_hello_world_config_config_2.yaml [hunk 3]
```diff
@@ -0,0 +1 @@
+sample_param: do a different thing
```
Comment (epec254) [inline M1_Sample_Code/2_hello_world_parameterized/configs/2_hello_world_config_config_2.yaml]: lets drop these files from the PR - they will get generated when running the code

File: M1_Sample_Code/2_hello_world_parameterized/configs/2_hello_world_config_config_1.yaml [hunk 4]
```diff
@@ -0,0 +1 @@
+sample_param: do this thing
```
Comment (epec254) [inline M1_Sample_Code/2_hello_world_parameterized/configs/2_hello_world_config_config_1.yaml]: lets drop these files from the PR - they will get generated when running the code

File: M1_Sample_Code/3_pdf_rag/3_rag_chain.py [hunk 5]
```diff
@@ -9,20 +9,21 @@
 # COMMAND ----------
 
 # Before logging this chain using the driver notebook, you need to comment out this line.
-dbutils.library.restartPython() 
+# dbutils.library.restartPython()
```
Comment (epec254) [inline M1_Sample_Code/3_pdf_rag/3_rag_chain.py]: Same comment on leaving this uncommented

File: M1_Sample_Code/4_rag_chain_w_conversation_history /4_rag_chain_w_conversation_history.py [hunk 6]
```diff
@@ -9,20 +9,21 @@
 # COMMAND ----------
 
 # Before logging this chain using the driver notebook, you need to comment out this line.
-dbutils.library.restartPython() 
+# dbutils.library.restartPython()
```
Comment (epec254) [inline M1_Sample_Code/4_rag_chain_w_conversation_history /4_rag_chain_w_conversation_history.py]: Same comment on leaving this uncommented

File: M1_Sample_Code/wheel_installer.py [hunk 7]
```diff
@@ -1,3 +1,3 @@
 # Databricks notebook source
-# MAGIC %pip install "PUT_RAG_STUDIO_WHEEL_HERE"
-# MAGIC %pip install "PUT_RAG_EVAL_SUITE_WHEEL_HERE"
+# MAGIC %pip install "https://ml-team-public-read.s3.us-west-2.amazonaws.com/wheels/databricks-rag-studio/679d2f69-6d26-4340-b301-319a955c3ebd/databricks_rag_studio-0.0.0a2-py3-none-any.whl"
```
Comment (epec254) [inline M1_Sample_Code/wheel_installer.py]: Lets' remove these wheels

**Review by epec254 — CHANGES_REQUESTED** [review]

**Pull request merged**